### ISSUE TITLE
[Question] Is there a way to "trick" websites and make them "think" Shumway is Adobe's Flash?

### ISSUE THREAD

**Issue by castarco** [mozilla/shumway#2317]
Hello.

It would be nice to be able to "fake" Adobe's Flash using Shumway. Currently there are many sites (like Kongregate) that block their contents when Adobe's Flash isn't installed, even if the user has installed Shumway on his/her browser.

Is it possible to configure shumway to advertise himself as adobe's flash? (like when people changes the User-Agent header sent by their web browsers).

Thank you for your efforts.

**Comment by tschneidereit** [comment]
Yes, that's obviously required. It will be possible once [bug 558184](
https://bugzilla.mozilla.org/show_bug.cgi?id=558184) is fixed in Firefox.
This is being worked on right now, so it will work fairly soon in Firefox
Nightly.

**Comment by jediminer543** [comment]
I've got a grease monkey script that stops sites searching for Flash, I named it Zoom (Hopefully someone gets the reference). It works in forcing the flash to load, but unfortunately shumway doesn't always work on the swf file.

https://gist.github.com/jediminer543/cb945d32e9761ef77b8d
